test/annexB/RegExp-leading-escape-BMP.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2009 the Sputnik authors.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+info: >
+    RegularExpressionFirstChar :: BackslashSequence :: \NonTerminator,
+    RegularExpressionChars :: [empty], RegularExpressionFlags :: [empty]
+es5id: 7.8.5_A1.4_T2
+es6id: 11.8.5
```

Comment:
The file is a test for the lexical grammar (11.8.5), not the regular expression grammar (21.2.1). And since the lexical grammar is not changed for B.1.4, the `es6id` field should be set to 11.8.5, shouldn't it?